core/themes/seven/js/script.js

```diff
@@ -0,0 +1,324 @@
+/**
+ * @file
+ * Responsive Admin tabs.
+ */
+
+(function ($) {
+"use strict";
+
+Backdrop.behaviors.responsivePrimaryTabs = {
+  attach: function(context, settings) {
+    var $primaryTabs = $('ul.tabs.primary');
+    var $tabsWrapper = $primaryTabs.parent();
+    var $tabs = $('li', $primaryTabs);
+    var responsiveTabs = false;
+    var previousWindowWidth;
+    var tabWidths = [];
+    var tabHeight;
+    var widestTabWidth = 0;
+    var expandControlWidth;
+    var activeTabNth = $('li.active', $primaryTabs).index();
+    var expandedTabsHeaderPadding = 0;
+    var defaultHeaderPadding = $('.l-header').css('padding-top');
```

Comment:
Are we still using this variable? It looks like other references to `.l-header` have been removed, per previous requests.